eatngo-core/src/main/kotlin/com/eatngo/search/domain/SearchStore.kt

```
@@ -0,0 +1,8 @@
+package com.eatngo.search.domain
+
+/**
+ * SearchStore
+ * 검색하는 가게의 정보를 담고 있는 클래스(MongoDB Document)
+ */
+class SearchStore {
+}
```

detekt (1.23.8)

[warning] The class or object SearchStore is empty.

(detekt.empty-blocks.EmptyClassBlock)